app/src/main/java/org/simple/clinic/summary/SummaryBloodPressureListItem.kt

```diff
@@ -16,18 +17,20 @@ import org.simple.clinic.R
 import org.simple.clinic.bp.BloodPressureMeasurement
 import org.simple.clinic.util.Just
 import org.simple.clinic.util.None
-import org.simple.clinic.util.Optional
 import org.simple.clinic.util.Truss
 import org.simple.clinic.util.Unicode
 import org.simple.clinic.widgets.UiEvent
+import org.simple.clinic.widgets.setPaddingBottom
+import org.simple.clinic.widgets.setPaddingTop
 import org.simple.clinic.widgets.setTextAppearanceCompat
 
 data class SummaryBloodPressureListItem(
     val measurement: BloodPressureMeasurement,
-    private val timestamp: RelativeTimestamp,
+    private val displayDaysTimestamp: RelativeTimestamp,
     val isEditable: Boolean = false,
     val showDivider: Boolean,
-    val displayTime: Optional<String>
+    val displayTime: String?,
+    val firstInGroup: Boolean
```

Comment:
These were named as `timestamps` before but now that we have two set of timestamps, one a day and the other a time, I wanted to rename. 
Since these parameters have values that are for displaying information into the text views directly, I used the word "display".